ui/raidboss/data/06-ew/trial/zodiark-ex.ts

```diff
@@ -19,6 +124,307 @@ const triggerSet: TriggerSet<Data> = {
       netRegexJa: NetRegexes.startsUsing({ id: '67EF', source: 'ゾディアーク' }),
       response: Responses.tankBuster(),
     },
+    {
+      id: 'ZodiarkEx Blue Tether',
```

Comment:
Blue (Purple?) is the cone? and red is the half arena?

Is it possible to add more differentiation between the two? It looks like right now that if you had a west red tether and an east blue tether that they would both say "east", but the safe spots for those are different.

Should the conal tether say "NE/SE" or "East Cone" or something else?